.gitmessage

```diff
@@ -0,0 +1,37 @@
+
+# [<tag>] <subject> (Max 80 char)
+# |<----   Preferably using up to 60 chars   --->|<------------------->|
```

Comment:
I would probably use smaller numbers see [here](https://chris.beams.io/posts/git-commit/).
The linux kernel developers recommend no more than [70-75 characters](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/submitting-patches.rst?id=bc7938deaca7f474918c41a0372a410049bd4e13#n664) for the title and the text too.